packages/e2e-tests/next-app/next.config.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
```

Comment:
Maybe best to leave next.config.js out. That would test serverless-next.js ability to generate a next.config.js for users.

Reply:
Yeah, I was planning to have different configurations in the future, this app would just be one of a few different apps (or we could also programmatically update the next.config.js for a single app).

For the first app, I could remove this.